.github/workflows/update-manifest.yml

```diff
@@ -0,0 +1,72 @@
+name: Update Manifest
+
+on:
+  workflow_call:
+    inputs:
+      overlay:
+        required: true
+        type: string
+    secrets:
+      token:
+        required: true
+
+jobs:
+  kustomize:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set versions
+        run: |
+          export C_VERSION="${GITHUB_REF_NAME#v}" 
+          export C_REVISION="${GITHUB_SHA}" 
+
+          echo "CONNECTORS_VERSION=${C_VERSION}" >> $GITHUB_ENV
+          echo "CONNECTORS_REVISION=${C_REVISION}" >> $GITHUB_ENV
+          echo "CONNECTORS_RELEASE=${C_VERSION}-${C_REVISION}" >> $GITHUB_ENV
+
+      - name: "Checkout cos-manifests project"
+        uses: actions/checkout@v2
+        with:
+          repository: bf2fc6cc711aee1a0c2a/cos-manifests
+          token: ${{ secrets.token }}
+
+      - name: "Update Kustomize overlay ${{ inputs.overlay }}"
+        uses: mikefarah/yq@master
```

Comment:
Wouldn't using a tag instead of master make the build more reproducible?

Reply:
This can probably removed entirely as yq is provided by the github virtual env anyway

Reply:
done